kernel/kernel_config-4.9.x

```diff
@@ -243,16 +243,13 @@ CONFIG_OPROFILE=y
 # CONFIG_OPROFILE_EVENT_MULTIPLEX is not set
 CONFIG_HAVE_OPROFILE=y
 CONFIG_OPROFILE_NMI_TIMER=y
-CONFIG_KPROBES=y
+# CONFIG_KPROBES is not set
```

Comment:
That would be controlled by `CONFIG_BPF_SYSCALL`, which we currently enable, so those will work.